src/assets/Default VPinballX.ini

```diff
@@ -18,13 +18,15 @@ HighDPI =
 VPRegPath = 
 PinMAMEPath = 
 PinMAMEWindow = 
+PinMAMEWindowRenderMode = 
```

Comment:
Some docs on what options are available and what the default is would be welcome.

Reply:
Accepting PRs!